src/main/java/net/sf/jabref/gui/entryeditor/EntryEditorTab.java

```diff
@@ -151,7 +153,8 @@ private void setupPanel(JabRefFrame frame, BasePanel bPanel, boolean addKeyField
                         false);
                 defaultHeight = 0;
             } else {
-                fieldEditor = new TextArea(field, null);
+                String prompt = (field.equals(FieldName.AUTHOR)) ? Localization.lang("Firstname Lastname and Firstname and Lastname and others") : "";
```

Comment:
I think, as our "normalize to bibtex name format" formatter will produce "Last, First and Last, First and other", the text should be in this form.